changelog.dd

```diff
@@ -71,6 +73,14 @@ assert(a.minIndex == -1);
 -------
 )
 
+$(LI $(LNAME2 ndslice-removal, `std.experimental.ndslice` has been
+deprecated.)
+    The synchronization between Phobos and Mir turned out to be a lot of work
+    with litte gain. Users of `std.experimental.ndslice` are advised to
+    switch to the upstream $(LINK2 https://github.com/libmir/mir-std,
+    mir package).
```

Comment:
It was changed to `mir-algorithm`. I don't think I understand now what we should put into `mir-std`, probably this would be only `traits` and `meta` and maybe compiler intrinsics.